### DISCUSSION TITLE
Can you close the menu from a d-bus command?

### DISCUSSION THREAD

**Discussion by Nickiel12** [Schneegans/Fly-Pie#190] [Ideas & Feature Discussions]
Hello, touchpad user here. I have been using Touche to call commands on gestures, and I already love pinching two fingers to summon the menu. However, I couldn't find a way to dismiss the menu if I had summoned it by accident, from a command. I would like to be able to pinch out with two fingers and touche to call a command that closes the menu.

I would like a D-Bus command that dismisses the menu that I can trigger from a command that I can trigger with Touche.

**Comment by Schneegans** [comment]
Thanks for the suggestion! This is indeed not possible right now, but should be easy to implement. I'll create an issue accordingly!

Until it's implemented, you can also long-press anywhere on the screen to dismiss the menu!

**Comment by Schneegans** [comment]
Here you go: #191.

**Comment by Schneegans** [comment]
There's now a `CancelMenu` D-Bus method in the `develop` branch!
